rust/tvm/src/ir/tir.rs

```diff
@@ -56,12 +56,35 @@ impl From<i32> for IntImm {
     }
 }
 
+impl From<i64> for IntImm {
+    fn from(i: i64) -> IntImm {
+        IntImm::new(DataType::int(64, 1), i)
+    }
+}
+
 impl From<i32> for PrimExpr {
     fn from(i: i32) -> PrimExpr {
         IntImm::from(i).upcast()
     }
 }
 
+impl Into<i64> for PrimExpr {
```

Comment:
I don't think we should provide `Into<i64>` for `PrimExpr`, because `Into` should only be used for infallible conversions.  Can we provide `TryInto<i64>` instead?